### PR TITLE
fix: self contained styles for tile grid component

### PR DESCRIPTION
**Pull request by hertweckhr1**
## Description
Self contained styles for tile grid component. Part of #136.

image
image

### CODE REVIEW

File: test/resources/layout-shared.css [hunk 1]
```diff
@@ -26,3 +26,35 @@ body {
...
+} 
+
+.fd-has-grid-column-span-2 {
+  grid-column: span 2;
+}
+
+.fd-has-background-color-accent-7 {
+  background-color: var(--fd-color-accent-7, #0cd7ed) !important;
+}
+
```
Comment (hertweckhr1) [inline test/resources/layout-shared.css]: ^^ Not sure if this is the best place for the fd-has-gride-row-span-2, fd-has-grid-column-span-2, and fd-has-background-color-accent-7 helper functions.

File: scss/components/tile-grid.scss [hunk 2]
```diff
@@ -17,6 +18,7 @@ $block: #{$fd-namespace}-tile-grid;
   position: relative;
 
   & > * {
+    @include fd-reset();
```
Comment (greg-a-smith) [inline scss/components/tile-grid.scss]: This _seems_ wrong.

File: test/templates/tile-grid/index.njk [hunk 3]
```diff
@@ -1,50 +1,37 @@
 {% extends "layout.njk" %}
 {% from "./../format.njk" import format %}
-{% from "../tile/component.njk" import tile, tile_content, tile_media, tile_actions %}
-{% from "../product-tile/component.njk" import product_tile, product_tile_content, product_tile_media %}
-{% from "../button/component.njk" import button %}
 {% from "./component.njk" import tile_grid %}
 {% from "../identifier/component.njk" import identifier %}
```
Comment (greg-a-smith) [inline test/templates/tile-grid/index.njk]: Identifier shouldn't be imported or used here.

**Comment by jbadan** [comment]
@greg-a-smith I'd be happier with less repetitive code. I'm 💯 on board with replacing with a generic style. I think now is the time to do the replacements we want and group it with all the other large breaking changes.

**Comment by hertweckhr1** [comment]
reworking for a layout-grid component.

**Pull request closed without merging**